apps/api-gateway/src/routes/buildings.controller.ts

```
@@ -0,0 +1,26 @@
+import { Body, Controller, Post, Version } from '@nestjs/common';
```

ESLint

[error] 'Version' is defined but never used.

(@typescript-eslint/no-unused-vars)